docs/make.jl

```diff
@@ -8,11 +9,12 @@ makedocs(;
     authors="Lilith Orion Hafner <lilithhafner@gmail.com> and contributors",
     repo="https://github.com/LilithHafner/Chairmarks.jl/blob/{commit}{path}#{line}",
     sitename="Chairmarks.jl",
-    format=Documenter.HTML(;
-        prettyurls=get(ENV, "CI", "false") == "true",
-        canonical="https://Chairmarks.lilithhafner.com",
-        edit_link="main",
-        assets=String[],
+    format=DocumenterVitepress.MarkdownVitepress(
+        repo = "https://github.com/LilithHafner/Chairmarks.jl",
+        devbranch = "main",
+        devurl = "dev",
+        deploy_url = "chairmarks.lilithhafner.com",
+        build_vitepress = false,
```

Comment:
The site is currently a 404 error, likely because there is not index.html or such at https://github.com/LilithHafner/Chairmarks.jl/tree/gh-pages/previews/PR10.

Maybe this will help?

```suggestion
```

Reply:
Yep! This fixes the 404 error, but the formatting is still broken.